pkg/model/components/addonmanifests/karpenter/iam.go

```diff
@@ -52,17 +52,19 @@ func addKarpenterPermissions(p *iam.Policy) {
 		// use existing kOps instance group launch templates
 		// "ec2:CreateLaunchTemplate",
 		"ec2:CreateFleet",
-		"ec2:RunInstances",
 		"ec2:CreateTags",
-		"iam:PassRole",
+		"ec2:RunInstances",
 		"ec2:TerminateInstances",
+		"ec2:DescribeAvailabilityZones",
 		"ec2:DescribeLaunchTemplates",
 		"ec2:DescribeInstances",
+		"ec2:DescribeInstanceTypeOfferings",
+		"ec2:DescribeInstanceTypes",
 		"ec2:DescribeSecurityGroups",
+		"ec2:DescribeSpotPriceHistory",
 		"ec2:DescribeSubnets",
-		"ec2:DescribeInstanceTypes",
-		"ec2:DescribeInstanceTypeOfferings",
-		"ec2:DescribeAvailabilityZones",
+		"iam:PassRole",
+		"pricing:GetProducts",
 		"ssm:GetParameter",
```

Comment:
Maybe sort them as in the generated policy below:
```
"ec2:CreateFleet",
"ec2:CreateTags",
"ec2:DescribeAvailabilityZones",
"ec2:DescribeInstanceTypeOfferings",
"ec2:DescribeInstanceTypes",
"ec2:DescribeInstances",
"ec2:DescribeLaunchTemplates",
"ec2:DescribeSecurityGroups",
"ec2:DescribeSpotPriceHistory",
"ec2:DescribeSubnets",
"ec2:RunInstances",
"ec2:TerminateInstances",
"iam:PassRole",
"pricing:GetProducts",
"ssm:GetParameter"
```

Reply:
I started doing that but apparently stopped half-way through :)